src/static/index.css

```diff
@@ -87,7 +92,6 @@ ul {
 
 ul li {
   margin: 0.5em 0;
-  cursor: pointer;
```

Comment:
Removed it from here and added it to a tags instead, otherwise messages appear clickable on the normal pages